rs/cli/src/main.rs

```diff
@@ -12,6 +12,26 @@ struct CargoCommands {
 enum SailsCommands {
     #[command(name = "sails", subcommand)]
     Sails(Commands),
+    #[command(name = "sails-js", subcommand)]
+    SailsJs(JsCommands),
```

Comment:
That won't work.
Only one command `sails` per `cargo-sails` executable.

Reply:
+1